frontend/tests/playwright/fixtures/product-page.ts

```diff
@@ -35,4 +35,58 @@ export class ProductPage {
       expect(sku).toMatch(/IF\d{3}-\d{3}(-\d{1,2})?$/);
       return sku;
    }
+
+   /**
+    * @description Locates and returns the current product price as a string with the format $###.## or $###
+    */
+   async getCurrentPrice(): Promise<number> {
+      const price = await this.page
+         .getByTestId('product-price-section')
+         .getByTestId('current-price')
+         .textContent();
+      expect(price).not.toEqual('');
+      expect(price).toMatch(/\$[0-9]+(\.[0-9]{1,2})?$/);
+      return parseFloat(price!.slice(1));
+   }
+
+   /**
+    * @description Locates and returns the original product price as a string with the format $###.## or $###
+    */
+   async getDiscountedPrice(): Promise<number> {
+      const price = await this.page
+         .getByTestId('product-price-section')
+         .getByTestId('compare-at-price')
+         .textContent();
+      expect(price).not.toEqual('');
+      expect(price).toMatch(/\$[0-9]+(\.[0-9]{1,2})?$/);
+      return parseFloat(price!.slice(1));
```

Comment:
I'm almost inclined to have this return `parseFloat(parseFloat(price!.slice(1)).toFixed(2));`

Reply:
If we are working with a price that has `.00` then we will need to repeat the `toFixed` call anyways since `parseFloat` will remove any decimal points that are `0`. Therefore, `parseFloat(parseFloat(price).toFixed(2))` is the same as `parseFloat(price)` where the price has `.00`.

![zero points](https://user-images.githubusercontent.com/22064420/212770287-1ad4d2d6-e42f-438e-868f-e7a7e8ab9845.png)

Reply:
Hmm, I am not sure if it is good practice, but we could add an alternative function to return the `price` as a `string`.

```typescript

 async getCurrentPriceAsString(): Promise<string> {
      return (await this.getCurrentPrice()).toFixed(2);
   }
      
```